src/vsr/replica.zig

```diff
@@ -7734,6 +7788,41 @@ pub fn ReplicaType(
             self.sync_dispatch(.idle);
         }
 
+        /// We have just:
+        /// - finished state sync,
```

Comment:
Maybe call it "supreblock sync"? It is a bit surprising that "state sync" doesn't actually sync the state itself, only meta info.